app/controllers/finance/ecf/statements_controller.rb

```diff
@@ -9,18 +9,9 @@ def show
         @ecf_lead_provider = lead_provider_scope.find(params[:payment_breakdown_id])
         @cpd_lead_provider = @ecf_lead_provider.cpd_lead_provider
 
-        @statements = Finance::Statement::ECF.where(cpd_lead_provider: @cpd_lead_provider).order(deadline_date: :desc).to_a
-        @statements.prepend Finance::Statement::ECF.new(cpd_lead_provider: @cpd_lead_provider, name: "Current")
```

Comment:
Maybe it's intended but should be noted, that removing this will remove the current statement. I don't see anywhere an equivalent of this. So this will result in one fewer statement in the ECF payment breakdown pages

Reply:
@asmega it is in here 
https://github.com/DFE-Digital/early-careers-framework/pull/1712/files#diff-ea0aa868734a91ebb8c48393d7521f7823427657f792995bb997677e9de41022R12